README.md

```diff
@@ -1,4 +1 @@
-paper-radio-group
-=================
-
-See the [component page](http://www.polymer-project.org/docs/elements/paper-elements.html#paper-radio-group) for more information.
+# paper-radio-group
```

Comment:
Alright. I've reverted it here, and I'll go through the other repos (soon-ish) and add it back.